bench/patches/v5/fix_letsencrypt_renew_frequency.py

```diff
@@ -0,0 +1,19 @@
+from bench.config.common_site_config import get_config
+from crontab import CronTab
+
+
+def execute(bench_path):
+	"""
+		This patch fixes a cron job that would renew letsencrypt certificate
+	"""
+
+    job_command = '/opt/certbot-auto renew --force-renewal -a nginx --post-hook "systemctl reload nginx"'
+	system_crontab = CronTab(user='root')
```

Comment:
Does this mean we now need sudo to create a new bench?

Reply:
No, this runs only if the letsencrypt command entry exists in cron